decidim-proposals/spec/events/decidim/proposals/evaluating_proposal_event_spec.rb

```diff
@@ -35,4 +35,41 @@
         .to include("The <a href=\"#{resource_path}\">#{translated(resource.title)}</a> proposal is being evaluated")
     end
   end
+
+  describe "notification digest mail" do
```

Comment:
See the comments left for `decidim-proposals/spec/events/decidim/proposals/accepted_proposal_event_spec.rb`